src/index.css

```diff
@@ -1,5 +1,123 @@
 body {
+  background: #fff5fe;
   margin: 0;
   padding: 0;
   font-family: sans-serif;
 }
+
+@keyframes beat{
+	to { transform: scale(1.05); }
+}
+
+svg {
+  overflow: inherit !important;
+}
+
+#organ_5, #organ_10, #organ_11, #organ_17, #organ_16 {
+  cursor: pointer;
+  fill-opacity: 0;
+  stroke-opacity: 0.2;
+  vector-effect: "non-scaling-stroke";
+  fill: #FF0000;
+  stroke: #FF0000;
+}
+
+#organ_5 {
+  animation: beat 1.3s infinite alternate;
+  fill-opacity: 0.6;
+}
+
+#pression-disable {
+  stroke: #004eff;
+  fill: transparent;
+  vector-effect: "non-scaling-stroke";
+  stroke-width: 5;
+  fill-opacity: 0.6;
+  stroke-opacity: 0.2;
+  cursor: pointer;
+}
+
+#organ_11, #cord {
+  cursor: pointer;
+  fill: rgb(245 181 173);
+  fill-opacity: 1;
+}
+
+ #cord:hover, #organ_17:hover, #organ_5:hover, #organ_10:hover, #organ_11:hover, #organ_17:hover, #organ_16:hover {
```

Comment:
This line is way too long. You should definitely use a class. Something like `.hoverable` or something.